package.json

```diff
@@ -13,15 +13,45 @@
     "url": "https://github.com/Techtonica/curriculum/issues"
   },
   "scripts": {
-    "validate-links": "remark -u validate-links ."
+    "precommit": "lint-staged",
+    "postcommit": "git update-index --again"
   },
   "devDependencies": {
+    "lint-staged": "^10.0.8",
+    "pre-commit": "^1.2.2",
+    "husky": "^1.2.1",
+    "prettier": "^2.0.1",
     "remark-cli": "^7.0.1",
     "remark-lint": "^6.0.5",
     "remark-lint-no-dead-urls": "^1.0.1",
     "remark-validate-links": "^9.0.1"
   },
   "remarkConfig": {
-    "plugins": ["remark-lint-no-dead-urls"]
-  }
+    "plugins": [
+      "remark-lint-no-dead-urls"
+    ]
+  },
+  "dependencies": {
```

Comment:
Why are these dependencies needed? I don't think any of them are used except for husky, which is already in `devDependencies`.